processor/deltatocumulativeprocessor/internal/streams/data.go

```diff
@@ -43,7 +43,7 @@ func Aggregate[D data.Point[D]](m metrics.Data[D], aggr Aggregator[D]) error {
 			return true
 		}
 		next.CopyTo(dp)
-		return false
+		return true
```

Comment:
Since this line was the cause of the bug, I'd be interested in reducing the confusion about whether `true` means `yes, continue iterating` or `yes, stop iterating`.

One approach would be to define constants CONTINUE=true and BREAK=false and using those in the loop bodies. I think readers would have much easier time following the loop logic.

Reply:
I think a comment should suffice.

Do we also need to flip the other boolean returned?

Reply:
If / when the golang RangeFunc proposal is merged, then we can use a native `range`, which will remove the confusion. But until then, I agree that a comment should be ok.